sdk/cognitivelanguage/azure-ai-language-conversations/azure/ai/language/conversations/_conversation_analysis_client.py

```diff
@@ -60,15 +60,14 @@ def send_request(
         # type: (...) -> HttpResponse
         """Runs the network request through the client's chained policies.
 
-        We have helper methods to create requests specific to this service in `azure.ai.language.conversations.rest`.
-        Use these helper methods to create the request you pass to this method.
-
+        >>> from azure.core.rest import HttpRequest
```

Comment:
to fix the pylint failures, since this is an autogenerated sdk, you can add `""azure-ai-language-conversations,` to `azure-sdk-for-python/eng/tox/allowed_pylint_failures.py`